src/frontends/pytorch/src/op/sort.cpp

```diff
@@ -39,6 +26,43 @@ OutputVector translate_sort(const NodeContext& context) {
                                                                   element::i64,
                                                                   stable));
     return topk->outputs();
+}
+
+OutputVector translate_sort(const NodeContext& context) {
+    num_inputs_check(context, 3, 4);
+    bool stable, descending;
+    int64_t dim;
+
+    if (context.get_input_size() == 4) {
+        stable = context.const_input<bool>(1);
+        dim = context.const_input<int64_t>(2);
+        descending = context.const_input<bool>(3);
+    } else {
+        stable = false;
+        dim = context.const_input<int64_t>(1);
+        descending = context.const_input<bool>(2);
+    }
+
+    return translate_sort_common(context, stable, dim, descending);
+};
+
+OutputVector translate_sort_fx(const NodeContext& context) {
+    // aten.sort.default(Tensor self, int dim=-1, bool descending=False) -> (Tensor values, Tensor indices)
+    num_inputs_check(context, 1, 3);
+    bool descending = false;
+    bool stable = false;
+    int64_t dim = -1;
+
+    if (!context.input_is_none(1)) {
+        dim = context.const_input<int64_t>(1);
+    }
+    if (!context.input_is_none(2)) {
+        descending = context.const_input<bool>(2);
+    }
+
+    auto topk_outputs = translate_sort_common(context, stable, dim, descending);
+    auto indices = context.mark_node(std::make_shared<ov::op::v0::Convert>(topk_outputs[1], element::i64));
```

Comment:
Do you really need to convert topk indices to i64 if you have TopK with `element::i64` created?